packages/server/lib/util/node_options.ts

```diff
@@ -27,6 +27,19 @@ export function needsOptions (): boolean {
   return true
 }
 
+/**
+ * Retrieve the current inspect flag, if the process was launched with one.
+ */
+function getCurrentInspectFlag (): string | undefined {
+  const flag = process.execArgv.find((v) => v.startsWith('--inspect'))
+
+  if (flag) {
+    return flag.split('=')[0]
+  }
+
+  return
```

Comment:
Stylistically do we like ternaries?

```suggestion
  return flag ? flag.split('=')[0] : undefined
```

Reply:
sure